plugin.video.twitch/resources/language/Danish/strings.xml

```diff
@@ -0,0 +1,61 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
You should move these strings.xml to strings.po
Not needed for inclusion, but the xml are deprecated and will stop working sooner or later